samples/grids/grid/data-searching/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "example-ignite-ui-react",
+  "description": "This project provides example of using Ignite UI for React components",
+  "author": "Infragistics",
+  "version": "1.4.0",
+  "license": "",
+  "homepage": ".",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "lint": "eslint ./src/**/*.{ts,tsx}"
+  },
+  "dependencies": {
+    "igniteui-react": "18.2.2-beta.0",
+    "igniteui-react-core": "18.2.2-beta.0",
+    "igniteui-react-datasources": "18.2.2-beta.0",
+    "igniteui-react-grids": "18.2.2-beta.0",
+    "igniteui-react-inputs": "18.2.2-beta.0",
+    "igniteui-react-layouts": "18.2.2-beta.0",
```

Comment:
you need to target proget packages for all IG packagese, e.g.
"@infragistics/igniteui-react-grids": "23.2.152"

Reply:
Why is this? The proget packages require some manual work (copy the files outside of the scope in node modules). Those which are in the public npm works fine.

Reply:
@HUSSAR-mtrela Also, this package does not contain the translated WC grid

Reply:
@mddragnev ignore me previous request per discussion with Graham and Stamen in Teams